mobile/src/libs/components/app/app.tsx

```diff
@@ -6,23 +6,27 @@ import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import SplashScreen from 'react-native-splash-screen';
 import { Provider as StoreProvider } from 'react-redux';
 
-import { Toast } from '#libs/components/components';
+import { Loader, Toast } from '#libs/components/components';
 import { useEffect } from '#libs/hooks/hooks';
 import { store } from '#libs/packages/store/store';
 import { Root as RootNavigation } from '#navigations/navigations';
 
+import { DELAY } from './libs/constants';
 import { styles } from './styles';
 
 const App: React.FC = () => {
   useEffect(() => {
-    SplashScreen.hide();
+    setTimeout(() => {
```

Comment:
Add a comment why this was added, something like this
```suggestion
// To prevent white screen after splash screen disappears
    setTimeout(() => {
```